tests/integration/values/values_helm_traces_load_balancing.yaml

```diff
@@ -0,0 +1,56 @@
+sumologic:
+  traces:
+    enabled: true
+    loadBalancing: true
```

Comment:
If you don't need logs at all (or metrics) think about disabling them altogether in this test case.

Then you could get rid of some of the values below

Reply:
Removed the logging exporter. Any other parts that can be safely removes?

Reply:
You can disable whole metrics and logs pipeline if you're not using them in this test.

```yaml
sumologic:
  logs:
    enabled: false

  metrics:
    enabled: false
```

And then the below added `fluent-bit` and `fluentd` sections can also be removed

Reply:
ping

Reply:
Setting `metrics.enable` to `false` has na unexpected consequences. Tests fail and on next run don't even start.

```
=== RUN   Test_Helm_Traces_Load_Balancing/traces/otelcol_deployment_is_ready

    helm_traces_load_balancing_test.go:47:
        	Error Trace:	helm_traces_load_balancing_test.go:47
        	            				env.go:442
        	Error:      	Received unexpected error:
        	            	timed out waiting for the condition
        	Test:       	Test_Helm_Traces_Load_Balancing/traces/otelcol_deployment_is_ready
=== RUN   Test_Helm_Traces_Load_Balancing/traces/otelgateway_deployment_is_ready
    helm_traces_load_balancing_test.go:73:
        	Error Trace:	helm_traces_load_balancing_test.go:73
        	            				env.go:442
        	Error:      	Received unexpected error:
        	            	timed out waiting for the condition
        	Test:       	Test_Helm_Traces_Load_Balancing/traces/otelgateway_deployment_is_ready
=== RUN   Test_Helm_Traces_Load_Balancing/traces/otelagent_daemonset_is_ready
I0315 16:59:35.083977 1130603 assess_funcs.go:219] DaemonSet "rel-2845692050-sumologic-otelagent" not yet fully ready
I0315 16:59:36.084736 1130603 assess_funcs.go:219] DaemonSet "rel-2845692050-sumologic-otelagent" not yet fully ready
I0315 16:59:37.084329 1130603 assess_funcs.go:219] DaemonSet "rel-2845692050-sumologic-otelagent" not yet fully ready
I0315 16:59:38.084087 1130603 assess_funcs.go:219] DaemonSet "rel-2845692050-sumologic-otelagent" not yet fully ready
I0315 16:59:39.082979 1130603 assess_funcs.go:219] DaemonSet "rel-2845692050-sumologic-otelagent" not yet fully ready
I0315 16:59:40.083652 1130603 assess_funcs.go:219] DaemonSet "rel-2845692050-sumologic-otelagent" not yet fully ready
...
```

Then durign next run
```
make test TEST_NAME="Test_Helm_Traces_Load_Balancing"
KIND_NODE_IMAGE="kindest/node:v1.22.7@sha256:1dfd72d193bf7da64765fd2f2898f78663b9ba366c2aa74be1fd7498a1873166" \
	go test -timeout 20m -v -count 1 -run=Test_Helm_Traces_Load_Balancing . -args
2022/03/15 17:02:05 Successfully read kind images spec
2022/03/15 17:02:05 Default kind image: kindest/node:v1.22.7@sha256:1dfd72d193bf7da64765fd2f2898f78663b9ba366c2aa74be1fd7498a1873166
2022/03/15 17:02:05 Supported kind images: [kindest/node:v1.22.7@sha256:1dfd72d193bf7da64765fd2f2898f78663b9ba366c2aa74be1fd7498a1873166 kindest/node:v1.21.10@sha256:84709f09756ba4f863769bdcabe5edafc2ada72d3c8c44d6515fc581b66b029c kindest/node:v1.20.15@sha256:393bb9096c6c4d723bb17bceb0896407d7db581532d11ea2839c80b28e5d8deb kindest/node:v1.19.16@sha256:81f552397c1e6c1f293f967ecb1344d8857613fb978f963c30e907c32f598467]
=== RUN   Test_Helm_Traces_Load_Balancing
Test_Helm_Traces_Load_Balancing 2022-03-15T17:02:26Z retry.go:91: Wait for all Kube Nodes to be ready
Test_Helm_Traces_Load_Balancing 2022-03-15T17:02:26Z node.go:33: Getting list of nodes from Kubernetes
Test_Helm_Traces_Load_Balancing 2022-03-15T17:02:26Z client.go:42: Configuring Kubernetes client using config file /tmp/kind-cluser-test-helm-traces-load-balancing-kubecfg1481543921 with context
Test_Helm_Traces_Load_Balancing 2022-03-15T17:02:26Z retry.go:103: Wait for all Kube Nodes to be ready returned an error: Not all nodes ready. Sleeping for 2s and will try again.
Test_Helm_Traces_Load_Balancing 2022-03-15T17:02:28Z retry.go:91: Wait for all Kube Nodes to be ready
Test_Helm_Traces_Load_Balancing 2022-03-15T17:02:28Z node.go:33: Getting list of nodes from Kubernetes
...
```

Reply:
Done.